### DISCUSSION TITLE
Get location config file

### DISCUSSION THREAD

**Discussion by BillyCottrell** [JanDeDobbeleer/oh-my-posh#4829] [Troubleshoot] [answered]
Hi,

So I setup oh my posh a while ago and I want to check my config file, but I am unable to find it...
I checked if a command is available. I thought I could use "oh-my-posh config edit" to edit the config file that way but even though usage says there are 3 commands "config [export|migrate|edit]" yet available commands don't have edit...

Any ideas on how to find the file or edit the file?

Thanks in advance!

**Comment by JanDeDobbeleer** [comment, marked as answer]
It's always stored in a variable called `POSH_THEME`.

**Reply by ASIJake** [reply]
This is great if you can run the terminal, but if you are recovering it from a separate drive where you only have the files, it would be nice to have a list of possible "defualt" folders to check. 🙂

**Reply by JanDeDobbeleer** [reply]
@csijake there are no default folders, it all depends where you added the file. So we can't help you with that.

**Reply by ASIJake** [reply]
Well, at least on certain Windows machines, I found that I was able to retrieve the value of `POSH_THEME` from `%localappdata%\oh-my-posh\omp.cache` (the environment variable was cached inside one of the serialized JSON payloads in that file). But YMMV.

The value then led me to the full file path where I had stored/customized my `*.omp.json` file.

Oh, and this was V2... the caching behavior in V3 might be different, too. 🙂

**Reply by ssbarnea** [reply]
Does this POPeven have a default value? I would like to use a default/implicit name instead of having to point it to a custom location.

**Reply by JanDeDobbeleer** [reply]
@ssbarnea what do you mean?

**Reply by ssbarnea** [reply]
I was hopping that creating a config file in a particular location on disk would make oh-my-posh load it, without having to pass the `--config` argument with specific path inside my shell init file. "aka implicit config filename".

**Reply by JanDeDobbeleer** [reply]
@ssbarnea yeah, had that idea once (which is what aliae uses), but decided against it as we have multiple file formats etc
